src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -552,17 +552,6 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 		DependsOnTargets="$(_OnResolveMonoAndroidSdks)">
 </Target>
 
-<!--
-  NOTE:
-  This target runs during Restore, and there is no $(RestoreDependsOn) property.
-  There appears to be no other way to do this other than use BeforeTargets.
--->
-<Target Name="_SetLatestTargetFrameworkVersionForPackageReference"
```

Comment:
What's the rationale for this change?  What does it fix?  What could it break?

Reply:
ah, moved to Legacy.  My bad.

Reply:
Details are in: https://github.com/xamarin/xamarin-android/pull/6174/commits/11289695e08b83429146e5648d81ab1453d9e0ab